website_event_sale_hide_ticket/models/event_event.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, models
+
+
+class Event(models.Model):
+    _inherit = "event.event"
+
+    @api.onchange("event_type_id")
+    def _onchange_type(self):
+        super(Event, self)._onchange_type()
+
+        if self.event_type_id.use_ticketing:
+            for type_ticket in self.event_type_id.event_ticket_ids:
+                ticket = self.event_ticket_ids.filtered(
```

Comment:
Here you directly access to `event_ticket_ids`, but in the previous line, you did: `self.event_type_id.event_ticket_ids`.
Is it normal?

Reply:
yes, event_type has tickets and event event too. Goal here is to sync a value from event_type to event_event